### PR TITLE
Bug Fix: #6721 global datasource overflow query panel

### LINKED ISSUE #6721: text of the name of global datasource overflows in the query panel
### What is the current behaviour?

When a datasource name is of higher length it overflows.

### How to reproduce the issue?

1. Go to Global datasource page
2. Add any datasource
3. Rename the datasource name from the modal
4. Go to the app
5. Click on the `+Add` button on the query panel at the bottom
6. Check the datasource name 

### Screenshot

Screenshot 2023-06-14 at 4 43 02 PM

### PR DESCRIPTION
**Pull request by nabeel001**
Fixes #6721 

I have made the required changes as suggested in https://github.com/ToolJet/ToolJet/pull/6748#pullrequestreview-1492437592
PFA the screenshot after the fix is made.

image

### CODE REVIEW

File: frontend/src/Editor/QueryManager/Components/DataSourceLister.jsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ import RunpyIcon from '@/Editor/Icons/runpy.svg';
 import AddIcon from '@assets/images/icons/add-source.svg';
 import { useTranslation } from 'react-i18next';
 import { getSvgIcon } from '@/_helpers/appUtils';
+import { ToolTip } from '../../../_components/ToolTip';
```
Comment (stepinfwd) [inline frontend/src/Editor/QueryManager/Components/DataSourceLister.jsx:8]: import { ToolTip } from '@/_components/ToolTip';

**Review by stepinfwd — CHANGES_REQUESTED** [review]

File: frontend/src/Editor/QueryManager/Components/DataSourceLister.jsx (hunk 1 above)
Comment (stepinfwd) [reply frontend/src/Editor/QueryManager/Components/DataSourceLister.jsx:8]: @nabeel001 suggested a change rest looks good

**Comment by nabeel001** [comment]
@stepinfwd, I have made the requested changes regarding the import statement.  @nakulnagargade, Kindly review the same and merge it.

File: frontend/src/Editor/QueryManager/Components/DataSourceLister.jsx [hunk 2]
```diff
@@ -63,10 +64,12 @@ function DataSourceLister({
             }}
           >
             {fetchIconForSource(source)}
-            <p data-cy={`${String(source.name).toLocaleLowerCase().replace(/\s+/g, '-')}-add-query-card`}>
-              {' '}
-              {source.name}
-            </p>
+            <ToolTip message={source.name}>
+              <p data-cy={`${String(source.name).toLocaleLowerCase().replace(/\s+/g, '-')}-add-query-card`}>
+                {' '}
```
Comment (stepinfwd) [inline frontend/src/Editor/QueryManager/Components/DataSourceLister.jsx:69]: can remove this

**Review by stepinfwd — CHANGES_REQUESTED** [review]
There are conflicts can you please resolve @nabeel001

File: frontend/src/Editor/QueryManager/Components/DataSourceLister.jsx (hunk 2 above)
Comment (nabeel001) [reply frontend/src/Editor/QueryManager/Components/DataSourceLister.jsx]: @stepinfwd, I have removed the line no. 69 as requested. @nakulnagargade, Kindly review the same.

**Comment by kavinvenkatachalam** [comment]
@nabeel001 Please resolve the conflicts

**Comment by tejakummarikuntla** [comment]
Hey @nabeel001 were you able to progress with the comments, it requires you to resolve the confits to take it forward. Kindly update.

**Comment by nabeel001** [comment]
this pull request is no longer required.

**Pull request closed without merging**